src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -262,15 +262,63 @@ For more details, see [External services](/docs/apm/apm-ui-pages/monitoring/exte
 
 ### JVMs
 
-When you drill into a specific JVM, the UI display charts driven by JVM metric data:
+The JVMs page for services instrumented with OpenTelemetry allows you to identify which service instances have unusual or unhealthy performance patterns. You can compare the key service health and Java Virtual Machine (JVM) metrics across any number of instances. 
+
+Here's a typical workflow:
+
+1. Find interesting JVMs using the table of summarized health metrics:
+    * Sort to find outliers.
+    * Use the filter bar to narrow down your search using tags.
+2. Select those interesting JVMs.
+3. Click to go to the second view, which displays the health and runtime metrics faceted by JVM.
+
+You can choose several service instances to compare, based on summaries of key metrics: response time, throughput, error rate, garbage collection time, and memory usage. Then, you can compare all those instances' JVM metrics collected by OpenTelemetry instrumentation using timeseries charts to spot problems.
+
+Review these additional topics about using the JVMs page:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="metric-details"
+    title="Runtime metric details"
+  >
+  When you drill into a specific JVM, the UI display charts driven by JVM metric data:
+
+    * JVM metrics follow the general [semantic conventions for runtime environment metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#runtime-environment-specific-metrics---runtimeenvironment).
+    * The Java specific runtime metrics are not well documented. The [implementation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics) is effectively the documentation and may be subject to change.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="config-steps"
+    title="How to ensure your data appears"
+  >
+  For your data to appear in this section, make sure it has the following:
+
+    * A unique `service.instance.id` attribute for rendering the list of JVMs
+    * An OpenTelemetry resource attribute `Service.instance.id`.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="metric-types"
+    title="Gauges versus counters"
+  >
+  Starting in OpenTelemetry Java agent 1.10.0, JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:
```

Comment:
Throughout this section, is "up down" ok or should it be "up-down"? I'm not sure based on the context, so I'll leave that decision to you.